### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#169] [Esercizi - Viaggi/Piazza]
![Es 6](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a1ce6174-8635-4bba-8066-5666cc5029dd)
($^2$) Quindi, più precisamente, la somma di due elementi di $\mathbb{Q}[\sqrt 2]$, visti come elementi di $\mathbb{R}$, è ancora un elemento di $\mathbb{Q}[\sqrt 2]$ e lo stesso è vero per il prodotto.

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/30710461-167c-42b2-b16b-9dd03920d74e)
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/685ab261-5e81-4faa-9132-2947a7611d56)

**Reply by CiottoloMaggico** [reply]
Allego secondo metodo per trovare l'inverso moltiplicativo utilizzando un sistema lineare, i risultati possono sembrare diversi anche se sono esattamente gli stessi.
Alpha e Beta trovati con il sistema coincidono con quelli del primo metodo infatti facendo la differenza Alpha1, Alpha2 e Beta1.Beta2 si ottiene 0.
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/e55c18d9-fc8b-4b39-84e2-5e99da01a75f)
